src/transformers/modeling_tf_utils.py

```diff
@@ -2313,6 +2313,13 @@ def save_pretrained(
             files_timestamps = self._get_files_timestamps(save_directory)
 
         if saved_model:
+            # If `torch_dtype` is in the config with a torch dtype class as the value, we need to change it to string.
```

Comment:
Nice comment :)

Reply:
A bit long though 😅 I think the first sentence is more than enough ;-)

Reply:
I will change it back to the first line lone :-).